nservicebus/outbox/index.md

```diff
@@ -79,8 +79,61 @@ In order to gradually convert an entire system from the DTC to the Outbox:
 WARNING: When verifying outbox functionality, it can sometimes be helpful to temporarily [stop the MSDTC Service](https://technet.microsoft.com/en-us/library/cc770732.aspx). This ensures that the Outbox is working as expected, and no other resources are enlisting in distributed transactions.
 
 
+## Message identity and idempotent processing
+
+If handlers are idempotent then outbox might not be required at all. When considering not using the outbox and implementing idempotent processing keep in mind that any generated message *must* apply the same message identifier if invoked more-then-once to have deduplication working on the infrastructure level or to solve this at the logical level by validating message data. [Read more on how to set a message identifier.](/nservicebus/messaging/message-identity.md)
```

Comment:
@ramonsmits the 2nd sentence here is unclear. The first part of it seems to make sense, but after "to have deduplication working..." it gets confusing, to the point where I'm not sure if the first part of the sentence even relates to it. 

Try breaking that sentence down into smaller pieces. Go as far as making bullet points of what you want to say. Then turn them into short sentences that each, in isolation, are completely clear. Read each sentence without the others around it. If the sentence still makes sense without the surrounding context, then it's in good shape. Eventually, if warranted, those can be joined together to make compound sentences.